src/lib/accessibility/Behaviors/Toolbar/toolbarBehavior.ts

```diff
@@ -2,6 +2,8 @@ import { Accessibility, FocusZoneMode } from '../../types'
 
 /**
  * @specification
```

Comment:
@description?  Or is there a difference?

Reply:
good catch. difference is that lines from specification are unit tested.